index.bs

```diff
@@ -949,10 +1041,10 @@ spec:html; type:element; text:link
 <pre class="biblio">
 {
   "COMCAST": {
-      "href": "http://arstechnica.com/tech-policy/2014/09/why-comcasts-javascript-ad-injections-threaten-security-net-neutrality/",
```

Comment:
Please don't mix substantive textual changes with whitespace changes like these.

Reply:
Reversed in latest commit. Understood. Still learning the W3C document review etiquette.